### PR TITLE
Issue#26   release min gw

### PR DESCRIPTION
**Pull request by oallaire**

### CODE REVIEW

File: library/src/hspec/mingw/mal_hspec_mingw_adc.h [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#ifndef HSPEC_MINGW_MAL_HSPEC_MINGW_ADC_H_
+#define HSPEC_MINGW_MAL_HSPEC_MINGW_ADC_H_
+
+#include "std/mal_error.h"
+#include "hspec/mal_hspec_cmn.h"
+#include "std/mal_stdint.h"
+
+mal_error_e mal_hspec_mingw_get_valid_adc_ios(mal_hspec_adc_e adc,const mal_hspec_gpio_s **ios, uint8_t *size);
+
+void mal_hspec_mingw_add_adc(mal_hspec_adc_e adc);
```
Comment (larosek) [inline library/src/hspec/mingw/mal_hspec_mingw_adc.h]: No method seems to be link to this declaration

Comment (oallaire) [inline library/src/hspec/mingw/mal_hspec_mingw_adc.h]: True, this is obsolete. I changed my mind half way threw coding this. It is normal, it is not used, I will remove this definition.

**Comment by larosek** [comment]
Other than the declaration in mal_hspec_mingw_adc.h that seem to be useless, everything seems good on my side.

**Comment by oallaire** [comment]
Good, I will commit this correction and then merge, thanks for review.

**Pull request merged**